Apps/package.json

```diff
@@ -3,17 +3,24 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "getNightly": "node scripts/getNightly.js"
+    "getNightly": "node scripts/getNightly.js",
+	"build" : "npx babel node_modules/mocha/mocha.js --out-file mocha.esm.js && npx babel node_modules/chai/chai.js --out-file chai.esm.js && npx rollup mocha.esm.js --file mocha.umd.js --format umd --name mocha && npx rollup chai.esm.js --file chai.umd.js --format umd --name chai",
```

Comment:
Where do these files end up? Are they just in the source folder?

Reply:
in Apps/ folder